packages/muelu/research/regionMG/src/SetupRegionHierarchy_def.hpp

```diff
@@ -475,39 +476,48 @@ void RebalanceCoarseCompositeOperator(const int maxRegPerProc,
 {
 #include "MueLu_UseShortNames.hpp"
   using CoordType = typename Teuchos::ScalarTraits<Scalar>::coordinateType;
+  using Teuchos::TimeMonitor;
+
+  RCP<TimeMonitor> tm = rcp(new TimeMonitor(*TimeMonitor::getNewTimer("ReblanceCoarseCompositeOperator: ")));
 
   RCP<Teuchos::FancyOStream> fos = Teuchos::fancyOStream(Teuchos::rcpFromRef(std::cout));
   fos->setOutputToRootOnly(0);
   *fos << "Rebalancing coarse composite operator." << std::endl;
 
-  //RCP<const Map> map = coarseCompOp->getMap();
-
   const int numPartitions = rebalanceNumPartitions;
 
   // We build a fake level
   Level level;
-  level.SetLevelID(1);
+  level.SetLevelID(3);
```

Comment:
Huh? Why?

Reply:
ah I didn't even see your review while I was writing mine :stuck_out_tongue: I had the same question.

Reply:
It is to start past the default repartition start level.

Reply:
That's a bit hacky, but ok. Maybe check that this doesn't appear anywhere in the output, since that could become quite confusing..